src/components/Header.js

```diff
@@ -1,77 +1,39 @@
 import React, { Component } from 'react';
+import SearchSVG from 'svg/Search';
+import Navmenu from 'components/Navmenu';
 
 class Header extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      menuIsOpen: false,
+    };
+  }
+
+  toggleMenu = () => {
+    this.setState({
+      menuIsOpen: !this.state.menuIsOpen,
+    })
+  }
 
   render() {
     return (
-
-      <header className="usa-header usa-header-basic" role="banner">
-        <div className="usa-nav-container">
-          <div className="usa-navbar">
-            <div className="usa-logo" id="basic-logo">
-              <em className="usa-logo-text">
-                <a href="/" title="Home" aria-label="Home">
-                  Welcome to Janis <span role="img" aria-label="peace sign">☮️</span>
-                </a>
-              </em>
+      <header className="coa-Header" role="banner">
+        <div className="wrapper">
+          <div className="row">
+            <div className="col-xs-6 coa-Header__menu">
+              <span onClick={this.toggleMenu} tabIndex="0">MENU</span>
+              <span className="coa-Header__text-spacer">|</span>
```

Comment:
Can the separator ‘|’ be a part of a :after content property so it’s not read as content? Or do we want it to be read as content?
Can we also make this more generic -- coa-text_spacer_vertical -- so we can use these all over? I have a feeling we'll have links listed like this in other places.

Reply:
sounds good, putting this in `helpers.scss`. Is that a good place for this type of thing?